Packs/FeedElasticsearch/Integrations/FeedElasticsearch/FeedElasticsearch.py

```diff
@@ -0,0 +1,351 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+'''IMPORTS'''
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+from elasticsearch_dsl import Search
+from elasticsearch_dsl.query import QueryString
+import requests
+import warnings
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+warnings.filterwarnings(action="ignore", message='.*using SSL with verify_certs=False is insecure.')
+
+HTTP_ERRORS = {
+    400: '400 Bad Request - Incorrect or invalid parameters',
+    401: '401 Unauthorized - Incorrect or invalid username or password',
+    403: '403 Forbidden - The account does not support performing this task',
+    404: '404 Not Found - Elasticsearch server was not found',
+    408: '408 Timeout - Check port number or Elasticsearch server credentials',
+    410: '410 Gone - Elasticsearch server no longer exists in the service',
+    500: '500 Internal Server Error - Internal error',
+    503: '503 Service Unavailable'
+}
+
+'''VARIABLES FOR FETCH INDICATORS'''
+FETCH_SIZE = 50
+API_KEY_PREFIX = '_api_key_id:'
+MODULE_TO_FEEDMAP_KEY = 'moduleToFeedMap'
+FEED_TYPE_GENERIC = 'Generic Feed (fill in configuration below)'
+FEED_TYPE_CORTEX = 'Cortex XSOAR Feed'
+FEED_TYPE_CORTEX_MT = 'Cortex XSOAR MT Shared Feed'
+
+
+class ElasticsearchClient:
+    def __init__(self, insecure=None, server=None, username=None, password=None, api_key=None, api_id=None,
+                 time_field=None, time_method=None, fetch_index=None, fetch_time=None, query=None):
+        self._insecure = insecure
+        self._proxy = handle_proxy()
+        # _elasticsearch_builder expects _proxy to be None if empty
+        if not self._proxy:
+            self._proxy = None
+        self._server = server
+        self._http_auth = (username, password) if (username and password) else None
```

Comment:
@DeanArbel should you check the api_key and api_id here. I think you always pass in a username and password which is set to a value.

Reply:
@glicht I don't have to check it, as explained in this comment: https://github.com/demisto/content/pull/5782#issuecomment-593069243